ext/oci8/oci8.c

```diff
@@ -127,6 +127,18 @@ ZEND_GET_MODULE(oci8)
 
 #include "oci8_arginfo.h"
 
+static PHP_INI_MH(OnUpdateOldCloseSemantics)
+{
+	bool *p = (bool *) ZEND_INI_GET_ADDR();
+	*p = zend_ini_parse_bool(new_value);
+
+	if (p) {
```

Comment:
Does it make sense to show the deprecation only when the ini setting is enabled, right?

Reply:
Yeah, that's how we usually do it.

Reply:
Oops, this should have been `*p`, not `p`.